React/Modules/RCTUIManager.m

```diff
@@ -1143,6 +1143,11 @@ - (void)batchDidComplete
   [self _layoutAndMount];
 }
 
+RCT_EXPORT_METHOD(playTouchSound)
```

Comment:
Adding dedicated managers is beyond my area of expertise. I found no other classes dealing with sound or audio in RN, so I think it's ok to do it this way until more sound business is needed.